coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
```

Comment:
coala does not pass any aspects to the bears. The bear writer has to decide the aspects the bear can handle while writing the bear. The aspects a bear handles does not change dynamically. 
IMO the bear collie toon methodology will have to be different as in one case we are given only the aspects and have to find the appropriate bears while in the other we are given the names of the bears themselves.